app/src/main/res/layout/activity_help_page.xml

```diff
@@ -101,30 +101,30 @@
                 android:layout_marginBottom="20dp">
 
 
-            <ImageView
+                <ImageView
+                    android:id="@+id/reject"
```

Comment:
Be careful when you merge! Either remove this or remove the listener added in the activity